free-2.md

```diff
@@ -46,6 +53,7 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+I think there are some slightly relevant DSLs in this domain. They are mostly concerning [business logic](http://www.drools.org/) and private industry rules. Drools seems like a pretty popular and effective business rule management system, and its interface seems extremely easy to use. I think because the domains are related but not exactly the same, it's difficutl to compare qualities. 
 
```

Comment:
This is really interesting and I think that things like Drools are good for helping businesses set standards. That being said, I think a language for government would run into a lot of problems, primarily from legislators opposing a computer-checked language designed to restrict them from being crafty in constructing their laws. It would be very difficult to get something like this approved by governmental groups.